Makefile

```diff
@@ -1,47 +1,57 @@
 DOCKER?=docker
+IMAGE?=
 
 # Base directories
 BASE_BUILD_DIR := $(CURDIR)/build
 REPRODUCIBLE_BUILD_DIR := $(CURDIR)/reproducible-build
+MEASUREMENTS_DIR := $(CURDIR)/measurements
 
-.PHONY: image-rbuilder image-bob image-base prepare-dirs clean tdx-poky
+.PHONY: help
+help: ## Display this help.
+	@awk 'BEGIN {FS = ":.*##"; printf "Usage:\n  make \033[36m<target>\033[0m\n"} /^[a-zA-Z_0-9-]+:.*?##/ { printf "  \033[36m%-15s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
 
-image-rbuilder: prepare-dirs
+.PHONY: image-rbuilder
+image-rbuilder: prepare-dirs ### Build rbuilder image, by default outputs to reproducile-build/artifacts-rbuilder and the measurements to measurements/
 	$(DOCKER) build -t yocto-builder:rbuilder --build-arg MANIFEST=tdx-rbuilder.xml $(REPRODUCIBLE_BUILD_DIR)
 	$(DOCKER) run --rm --env-file env_files/rbuilder_yocto_build_config.env \
 		-v $(REPRODUCIBLE_BUILD_DIR)/artifacts-rbuilder:/artifacts \
 		-v $(BASE_BUILD_DIR)/rbuilder:/build \
 		yocto-builder:rbuilder
 	chmod 0755 $(BASE_BUILD_DIR)/rbuilder $(REPRODUCIBLE_BUILD_DIR)/artifacts-rbuilder
+	MEASUREMENTS_DIR=$(MEASUREMENTS_DIR) IMAGE=$(REPRODUCIBLE_BUILD_DIR)/artifacts-rbuilder/ $(MAKE) gen-measurements
```

Comment:
do you mean by IMAGE, the image directory here? If not, you need to specify the name of the image, such as ` cvm-image-azure-tdx.rootfs.wic.vhd`

Reply:
Also in this makefile, there is no gen-measurements target if I am not wrong. 
So why are we adding this here?